doc/filters/html_to_markdown.rst

```diff
@@ -64,3 +63,23 @@ You can also use the filter on an included file:
                 }
             }
         });
+
+``html_to_markdown`` is just a frontend; the actual conversion is done by one of
+the following compatible libraries, from which you can choose:
+
+* [erusev/parsedown](https://github.com/erusev/parsedown)
+* [thephpleague/html-to-markdown](https://github.com/thephpleague/html-to-markdown)
+* [michelf/php-markdown](https://github.com/michelf/php-markdown)
+
+Depending on the library, you can also add some options by passing them as an argument
+to the filter. Example for ``league/html-to-markdown``:
+
+.. code-block:: twig
+
+    {% apply html_to_markdown({hard_break: false}) %}
+        <html>
+            <h1>Hello!</h1>
+        </html>
+    {% endapply %}
+
+.. note::
```

Comment:
why an empty note here ?

Reply:
Sorry - copy-paste leftover...  
Deleted now.

Reply:
GitHub hangs - I hope it got throuh...